### PR TITLE
feat: opentelementry support (update of #14668)

### PR DESCRIPTION
**Pull request by Candinya**
## Involved Issue / 该 PR 相关 Issue

Close #14668

## Example for the Proposed Route(s) / 路由地址示例

```routes
NOROUTE
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [x] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

Basically an updated version of #14668 , with some mentioned errors fixed.

### CODE REVIEW

File: lib/utils/otel/trace.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
+
+const provider = new BasicTracerProvider({
+    resource: new Resource({
+        [SEMRESATTRS_SERVICE_NAME]: 'rsshub',
+    }),
+});
+
+const exporter = new OTLPTraceExporter({
+    // optional OTEL_EXPORTER_OTLP_ENDPOINT=https://localhost:4318
```
Comment (TonyRL) [inline lib/utils/otel/trace.ts:13]: I think this can be updated thru env variable, isn't it?

Comment (incubator4) [reply lib/utils/otel/trace.ts:13]: > I think this can be updated thru env variable, isn't it?

due to https://www.npmjs.com/package/@opentelemetry/exporter-trace-otlp-http 
this env would be auto load by this package
